projects/batfish-common-protocol/src/org/batfish/datamodel/Interface.java

```diff
@@ -173,17 +187,22 @@ else if (name.startsWith("TenGigabitEthernet")) {
          return InterfaceType.PHYSICAL;
       }
       else if (name.startsWith("Tunnel")) {
-         return InterfaceType.VPN;
```

Comment:
This change is only OK if Tunnel interfaces are definitely not used for IPSEC VPNs. Please answer whether this is the case.